src/Apps/Feature/Components/FeatureFeaturedLink.tsx

```diff
@@ -50,40 +50,40 @@ export const FeatureFeaturedLink: React.FC<
               lazyLoad
             />
           </ResponsiveBox>
-        </Figure>
-      )}
-
-      {!img && title && (
-        <Text variant="lg-display" color="mono100" my={2}>
-          <RouterLink to={href}>{title || "—"}</RouterLink>
-        </Text>
-      )}
-
-      <Flex flexDirection={size === "large" ? ["column", "row"] : "column"}>
-        <Spacer y={1} />
+        )}
 
-        {subtitle && (
-          <Text variant="xs" fontWeight="bold">
-            {subtitle}
+        {!img && title && (
+          <Text variant="lg-display" color="mono100" my={2}>
+            {title || "—"}
           </Text>
         )}
 
-        <Text variant="lg-display" mt={0.5} lineClamp={3}>
-          {title}
-        </Text>
+        <Flex flexDirection={size === "large" ? ["column", "row"] : "column"}>
+          <Spacer y={1} />
 
-        {description &&
-          (size === "full" ? (
-            <HTML
-              variant="lg-display"
-              color="mono60"
-              mt={1}
-              html={description}
-            />
-          ) : (
-            <HTML variant="sm" color="mono60" mt={0.5} html={description} />
-          ))}
-      </Flex>
+          {subtitle && (
+            <Text variant="xs" fontWeight="bold">
+              {subtitle}
+            </Text>
+          )}
+
+          <Text variant="lg-display" mt={0.5} lineClamp={3}>
+            {title}
+          </Text>
+
+          {description &&
+            (size === "full" ? (
+              <HTML
+                variant="lg-display"
+                color="mono60"
+                mt={1}
+                html={description}
+              />
+            ) : (
+              <HTML variant="sm" color="mono60" mt={0.5} html={description} />
```

Comment:
Sorry just thought of a wrinkle. These descriptions contain HTML and so they may also have links. What we could do is a parallel anchor tag that's then absolutely positioned but avoids nesting the description.

Reply:
Think I have a solution:

1. Un-nesting the html desc from the parent still leaves it within the outer flex container and sized appropriately, while allowing the content to have its own links.
2. Adjusting the hover-color to just target the router link gets us the effect we like, with the bonus of working on title-hover, which is more intuitive

Reply:
Cool, that works/makes sense!